custom/icds_reports/migrations/0187_update_system_usage_report_view.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you check if the migration is on the right track. As in it has not conflict with master?
Also we actually name the new migration we add incrementing the count in the file name

Reply:
I just looked up all the tabular reports and added awc site code in all the reports which has awc name column.
(I added incentive report later. So, I missed adding it in the description)
(Also regarding the migration files, I will update it as soon as this pr: https://github.com/dimagi/commcare-hq/pull/27201 is approved)